deployment/ccip/changeset/v1_6/cs_add_new_chain_e2e.go

```diff
@@ -339,11 +343,20 @@ func addCandidatesForNewChainLogic(e deployment.Environment, c AddCandidatesForN
 				SkipChainConfigValidation: true,
 			},
 		},
+		// use currentDonId here
 		DonIDOverrides: map[uint64]uint32{c.NewChain.Selector: donID},
 	})
 	if err != nil {
 		return deployment.ChangesetOutput{}, fmt.Errorf("failed to run SetCandidateChangeset on home chain: %w", err)
 	}
+
+	// Claim donID using donIDClaimer at the end of the changeset run
```

Comment:
Maybe we should add an option to the config of this changeset to sync the claimer with an offset before claiming the next don ID?

Reply:
yea that'll be helpful. Added